pkg/ko/publish/daemon.go

```diff
@@ -47,14 +48,27 @@ func (d *demon) Publish(img v1.Image, s string) (name.Reference, error) {
 	if err != nil {
 		return nil, err
 	}
+
+	if len(d.tags) == 0 {
```

Comment:
You can avoid this special casing if you just make the default value for `--tags` be `[]string{"latest"}`.

(The third parameter in `StringSliceVar` allows you to specify a default.)

Reply:
thanks, changed this.